### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by simonepri**
### Changes
Versions tested where inconsistent across `appveyor` and `travis`

Now tests are runned for the following versions:
- "9"
- "8"
- "7"
- "6"
- "5"
- "4"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
-   - "6"
-   - "4"
+  - '9'
+  - '8'
+  - '7'
+  - '6'
+  - '5'
+  - '4'
+  - '0.12'
+  - '0.10'
```
Comment (soyuka) [inline .travis.yml]: I want to start testing from node 4, I mean LTS is 8 and < 4 should not be supported anymore.

File: .travis.yml [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
+  directories:
+  - node_modules
+install:
+  - node --version
+  - npm --version
+  - npm install
+script:
+  - node --version
+  - npm --version
+  - npm test
```
Comment (soyuka) [inline .travis.yml]: You could get rid of `install` and `script` as travis will already do all this by default

Comment (simonepri) [reply .travis.yml]: Is just to make `.travis.yml` look exacly like `appveyor.yml`.
I call it consistency. But yeah is reduntant.

**Comment by simonepri** [comment]
Sorry for the too many commits. I was doing this from the web editor.
When you merge it remember to squash the commits.

**Comment by soyuka** [comment]
https://github.com/soyuka/pidusage/commit/dbb37db5f9f449932cbd83c02227c68f661d4f37

**Comment by soyuka** [comment]
thanks @simonepri

**Comment by simonepri** [comment]
🔝

**Comment by simonepri** [comment]
@soyuka 
Referring to this:
https://github.com/soyuka/pidusage/commit/2ee72aa829798e713675feaa72cb4f36c3cb89ae
We should keep node 6 and remove node 9.
LTS versions of node are the ones that are even.

**Pull request closed without merging**